content/authorization/sign-in-with-livechat/index.mdx

```diff
@@ -148,166 +129,210 @@ If they are already logged in, you will immediately receive valuable information
+  - **code_challange_method** - code challenge method, use `S256` or `plain`, default: `S256`
 
-- **client_id** – obtained from the [Developers Console](https://developers.livechatinc.com/console/) when you create your app.
+<aside class="notice">
+  Use of PKCE extension is required when JavaScript application wants to acquire authorization code and exchange it for tokens.
+</aside>
 
-- **response_type** – (optional, defaults to `token`) Defines the type of the response that you will receive in `onIdentityFetched` callback. Two options are supported:
+#### **instance.iframe(options = {})** - Returns `Iframe` object instance built on top of sdk.
 
-  - **token** (default) – the response will include `access_token` that can be immediately used for calling LiveChat APIs. Best suitable for client-side apps.
-  - **code** – the response will include `code` that can be exchanged for `access_token` and `refresh_token`. Best suitable for backend apps that authorize a user only once and refresh the `access_token` themselves from that moment.<br/><br/>Read more about client-side and backend apps in <a href="https://developers.google.com/identity/protocols/OAuth2#webserver">Google OAuth tutorial</a>.
+#### **instance.popup(options = {})** - Returns `Popup` object instance built on top of sdk.
 
-- **onIdentityFetched(error, data)** – the callback executed when user's identity is fetched. The callback will include either `error` or `data` object depending on the current user authorization status. You will find the detailed documentation in <a href="#response-format">Response format</a> section.
+#### **instance.redirect(options = {})** - Returns `Redirect` object instance built on top of sdk.
 
-- **linkLabel** – (optional, defaults to empty string) allows to change default _Sign in with LiveChat_ label with custom text.
+#### **instance.authorizeURL(options = {}, flow = '')** - Creates authorization URL for given flow and parameters.
 
-- **popupRoute** – (optional, defaults to empty string) it lets you change the default popup endpoint. Possible values are:
-  - `''` (default) – an empty string, which will show the sign-in view.
-  - `'signup'` – it will point a user directly to the first step of the signup.
+#### **instance.verify(authorizeData)** - This function verifies if redirect transaction params are valid, returns `Transaction` if valid, `null` otherwise.
 
 <aside class="notice">
-  <code>accountsSdk</code> object exposes only one method: <code>init()</code>.
-  All other methods must be called by the object instance returned by the{" "}
-  <code>init()</code> method.
+  `options` objects in all methods of `AccountsSDK` object is used to overwrite data provided upon initialization.
 </aside>
 
-### **instance.updateOptions()**
+### **Iframe**
 
-> Example `updateOptions()` method usage:
+Class responsible for acquiring user authorization data through `iframe`.
```

Comment:
I wonder if we could describe this use a little bit more. As a LiveChat developer we know how it works, but external developers might need more info.

Reply:
Yeah, adding a bit more context would be great.
cc @Adimus11

Reply:
Okay, since Iframe is deprecated, this one can be resolved.